hr_attendance_calendar/models/hr_attendance_day.py

```diff
@@ -125,8 +125,11 @@ def _compute_due_hours(self):
                 continue
 
             # Contract
+            # sum the due hours from today attendance prescription for the
+            # employee. These attendance are selected in
+            # update_calendar_attendance() which is called upon
+            # attendance_day creation.
```

Comment:
I didn't really understand your comment... where is the method update_calendar_attendance?

Reply:
Further down in the file